### PR TITLE
sean completed react form controls

### PR DESCRIPTION
**Pull request by Sean-Hughes-1**
formcontrols

### CODE REVIEW

File: react-form-controls/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    this.setState({ pass: e.target.value });
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label htmlFor='uname'>User name: </label><br />
+        <input type='text' id='uname' name='uname' onChange={this.handleUserNameChange} value={this.state.username} /><br />
+        <label htmlFor='pass'>Password: </label><br />
+        <input type='text' id='pass' name='pass' onChange={this.handlePasswordChange}/><br /><br />
```
Comment (Zhiwutian) [inline react-form-controls/src/index.jsx:35]: don't forget to include the `value` attribute on this input to make sure its a controlled component

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update!

File: react-form-controls/src/index.jsx (hunk 1 above)
Comment (Sean-Hughes-1) [reply react-form-controls/src/index.jsx]: updated

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**